definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-order-nonprod.json

```diff
@@ -75,6 +75,8 @@
     "PageLabel": "General order",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/create-general-order/draft",
```

Comment:
This needs to be on the 1st field of the page (i.e. PageFieldDisplayOrder:1) otherwise it doesn't trigger 😅

Reply:
:o 

Good to know. Thanks :)

Reply:
updated